tests/utils.py

```diff
@@ -92,6 +94,12 @@ def get_contract_class(contract, is_path=False):
     return contract_class
 
 
+def get_class_hash(contract_name, is_path=False):
+    """Return the class_hash for a given contract."""
+    contract_class = get_contract_class(contract_name, is_path)
+    return compute_class_hash(contract_class=contract_class, hash_func=pedersen_hash)
+
```

Comment:
is this needed? if i understand correctly, [it was an environment issue right](https://github.com/OpenZeppelin/cairo-contracts/issues/529#issuecomment-1346071651)?

Reply:
The linked issue was an environment issue, but there's another issue. Nile's `get_contract_class` still expects the contracts to be already compiled in `artifacts` prior to testing (which would require adding `nile compile tests/mocks/` in the CI); whereas, Contracts's `get_contract_class` compiles the contracts as part of the testing flow.

I prefer the latter, but we can easily just use Nile's `get_contract_class` and `get_class_hash` in Contracts if it's favored

Reply:
we already have a [compile step in the CI](https://github.com/OpenZeppelin/cairo-contracts/blob/main/tox.ini#L30), but I'm not sure this PR fully replaces it. no strong opinions in using this or nile's

Reply:
> we already have a [compile step in tox](https://github.com/OpenZeppelin/cairo-contracts/blob/main/tox.ini#L30),

Correct, but it doesn't compile the mocks^ so when `signer.declare_class` is used (which checks the `artifacts` dir), it won't discover the mock contracts